src/ec/packages/ec-component-description-list/__snapshots__/description-list.test.js.snap

```diff
@@ -47,6 +47,43 @@ exports[`EC - Description list Default renders correctly 1`] = `
 </dl>
 `;
 
+exports[`EC - Description list Default renders correctly in the horizontal variant 1`] = `
+<dl
+  class="ecl-description-list"
```

Comment:
this is not the horizontal version tested here (missing css class)

Reply:
Oh, true, i assumed that the variant was coming from the spec package without checking it...